etc/compile/run_go_cmds.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# This file actually builds Pachyderm binaries ('pachd' and 'worker') and is
+# called by etc/compile/compile.sh inside the Pachyderm build container, but it
+# run as an unprivileged user with the host caller's user ID, instead of as
+# root. This prevents 'go build' from littering the pachyderm git directory
+# with root-owned files
+
+set -Eex
+
+env
```

Comment:
likewise, do we want to print out env here?

Reply:
Fixed